src/Native/Runtime/unix/PalRedhawkUnix.cpp

```diff
@@ -1014,14 +1014,6 @@ extern "C" Int32 _stricmp(const char *string1, const char *string2)
     return strcasecmp(string1, string2);
 }
 
-// Given the OS handle of a loaded module, compute the upper and lower virtual address bounds (inclusive).
```

Comment:
We are getting the bounds in the custom section, but that's not really the reason why this API is not needed.

The only use of this API was a sanity check to ensure that nobody else will handle hard crash in unmanaged runtime code. I do not think we need this sanity check on Unix.